src/app/pages/connect-page/_services/connect.model.service.ts

```diff
@@ -30,11 +30,25 @@ export class ConnectModelService {
         })
     }
 
-    fetchQRCodeData(){
+    fetchQRCodeData() {
         return  this._connectApiService.getQRCodeData();
     }
 
     getAllConnections() :any[] {
       return this.model;
     }
+
+    removeConnection(connectedWithUserId: number) {
+        return new Promise((resolve, reject) => {
```

Comment:
No need to return a new promise here, you can return the one from the apiService call. Likewise, above in your init() call..